mod/message.php

```diff
@@ -148,11 +148,7 @@ function message_content(App $a)
 
 		$cmd = $a->argv[1];
 		if ($cmd === 'drop') {
-			$r = q("DELETE FROM `mail` WHERE `id` = %d AND `uid` = %d LIMIT 1",
-				intval($a->argv[2]),
-				intval(local_user())
-			);
-			if ($r) {
+			if (dba::delete('mail', ['id' => $a->argv[2]])) {
```

Comment:
There's no previous ownership test, I believe the `uid` condition should be kept in place.